incubator/exporter-otlp.spec/src/main/scripts/io/aklivity/zilla/specs/exporter/otlp/application/event/server.rpt

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+accept "zilla://streams/app0"
+    option zilla:window 8192
+
+accepted
+connected
+
+accept "http://localhost:4318/v1/logs"
+accepted
+connected
+
+read http:method "POST"
+read http:version "HTTP/1.1"
+read http:header "Host" "localhost:4318"
+read
+'{'
+    '"resourceLogs":['
+        '{'
+            '"resource":{'
+                '"attributes":['
+                    '{'
+                        '"key":"service.namespace",'
+                        '"value":{'
+                            '"stringValue":"example"'
+                        '}'
+                    '}'
+                ']'
+            '},'
+            '"scopeLogs":['
+                '{'
+                    '"scope":{'
+                        '"name":"OtlpLogsSerializer",'
+                        '"version":"1.0.0"'
+                    '},'
+                    '"logRecords":['
+                        '{'
+                            '"timeUnixNano":42000000,'
+                            '"observedTimeUnixNano":42000000,'
+                            '"traceId":"8000000000000001",'
+                            '"spanId":"100000004",'
```

Comment:
Where did the values for `traceId` and `spanId` come from?
Perhaps we can leave these unset (if permitted) since we don't have a valid `otlp` trace or span to associate them with just yet.

Reply:
both are optional, removed for now